### PR TITLE
Upstream updates

### PR DESCRIPTION
**Pull request by mcdonnnj**
# 

## 🗣 Description

This PR pulls in the latest upstream to bring this repository into the **_future_**.

## 💭 Motivation and Context

This repo is consuming [apb](https://github.com/cisagov/action-apb) dispatches without the ability to process them. Also, it is out of date.

## 🧪 Testing

Automated tests pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -15,40 +18,52 @@ jobs:
...
-            terraform-${{ env.TERRAFORM_VERSION }}"
+          go-version: '1.14.2'
+      - name: Store installed Python version
+        run: |
+          echo "::set-env name=PY_VERSION::"\
+          "$(python -c "import platform;print(platform.python_version())")"
+      - name: Store installed Go version
+        run: |
+          echo "::set-env name=GO_VERSION::"\
+          "$(go version | sed 's/^go version go\([0-9.]\+\) .*/\1/')"
```
Comment (felddy) [inline .github/workflows/build.yml:35]: `set-env` changes needed.  Now or later...

**Review by felddy — APPROVED** [review]
Nice!  🍨 
Had a comment about `set-env` in workflows.  But I assume that'll come along with Lineage later.

File: .github/workflows/build.yml (hunk 1 above)
Comment (mcdonnnj) [reply .github/workflows/build.yml:35]: It should pull them down when https://github.com/cisagov/skeleton-tf-module/pull/35 is merged.

Comment (mcdonnnj) [reply .github/workflows/build.yml:35]: I'd rather let Lineage generate a PR to merge them automatically vs possibly introducing issues with `git` history.

Comment (jsf9k) [reply .github/workflows/build.yml:35]: `git` will sort it out, but I'm fine either way.

**Comment by felddy** [comment]
- [x] ⚠️ Let's get some branch protection all up in here.

**Review by jsf9k — CHANGES_REQUESTED** [review]
I have some changes to make here before this can be merged.  Stand by!

**Review by jsf9k — APPROVED** [review]
This looks great.  Thanks for waiting while I did a little more modernization.

**Review by dav3r — APPROVED** [review]
Solid! 🦊

**Comment by mcdonnnj** [comment]
> 
> 
> This looks great. Thanks for waiting while I did a little more modernization.

Thanks for taking care of that!

**Pull request merged**